examples/07-Train-a-third-party-model-using-the-Merlin-Models-API.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Can we rename the notebooks and all reference to <code> traditional ml models</code> . We are leveraging 3rd party libraries, it sounds not well. TensorFlow is a 3rd party library as well.</p><p> </p><p>Can you have the same order as in the notebook:  Change <code>xgboost</code>, <code>implicit</code> and <code>lightFM</code> to XGBoost, lightFM, implicit</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/629/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
done